src/auxiliary_functions.py

```diff
@@ -0,0 +1,63 @@
+
+from models import ClassificationResponse, Metadata, OtherProvidersClassifications
+from errors import ApkAnalyzerError
+from apk_analyzer import ApkAnalyzer
+from classifier import Classifier
+import time
+import logging
+import zipfile
+from io import BytesIO
+import os
+from dotenv import load_dotenv
+
+if os.getenv("DOCKER_ENV"):
+    load_dotenv(".docker.env")
+else:
+    load_dotenv(".env")
+
+CSV_PATH = os.getenv('CSV_PATH_FROM_SRC')
+
+class Auxiliary:
+    def __init__(self, apk_analyzer, redis_cache):
+        self.apk_analyzer = apk_analyzer
+        self.redis_cache = redis_cache
+
+    @staticmethod
+    def tracer(func):
+        def wrapper(*args):
+            logging.debug(f"{func.__name__} function")
+            before = time.time()
+            result  = func(*args)
+            logging.debug(f"====================> Execution time: {time.time() - before} seconds")
+            return result
+        return wrapper
+
+
+    def get_file_details(self, file_content):
```

Comment:
what is the idea of this function? why is it here?? 
utils functions should be generic, this function is very not generic and should be under the module that uses it